### PR TITLE
adding darts to the team-config.yml

### PR DESCRIPTION
**Pull request by scott-robertson1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-26

### Change description ###
adding darts to the team config file 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: team-config.yml [hunk 1]
```diff
@@ -141,4 +141,12 @@ makeaplea:
...
\ No newline at end of file
+    application: makeaplea
+darts:
+  team: "Darts Modernisation"
+  namespace: "darts"
+  slack:
+    contact_channel: "#darts-devs"
+    build_notices_channel: "#darts-builds"
+  tags:
+    application: DARTS
```
Comment (timja) [inline team-config.yml:152]: can you confirm this tag with `Bruce Rawstorne` on slack please?

I suspect at a minimum it would be lower case

Comment (jackmaloney) [reply team-config.yml:152]: It's an acronym and other similarly namedprojects within this file are in uppercase, FYI

Comment (timja) [reply team-config.yml:152]: the only other one is IAC-Fees, often these acronym's get expanded, this is what goes into finance reports so just need to make sure Bruce is happy with the tags.

If you can ping him and ask him about it i can approve for now and then you can update it?

Comment (scott-robertson1) [reply team-config.yml:152]: Have sent him a message, but currently showing as offline on slack.

Comment (scott-robertson1) [reply team-config.yml]: no response, so discussed with @jackmaloney and changed it to lower case

**Review by timja — APPROVED** [review]

File: team-config.yml (hunk 1 above)
Comment (rawgoose) [reply team-config.yml]: Hi
Would be good to talk about what is being built for "darts" as might be one of those that we keep uppercase as everyone know it as that.  Let's chat on Slack.

**Pull request merged**